src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -870,5 +870,23 @@ public async Task EvaluatePropertyThatThrows()
                props = await GetObjectOnFrame(frame, "this");
                CheckNumber(props, "a", 11);
            });
+
+        [ConditionalFact(nameof(RunningOnChrome))]
+        public async Task EvaluateMethodAndCheckIsNotPausingOnBreakpoint()
+        {
+            var waitForScript = WaitForWarningMessage("console.warning: MONO_WASM: Adding an id (0) that already exists in commands_received");
```

Comment:
What does this message mean, and why are we waiting to see that we don't get it?

Reply:
This message means that we are paused in a breakpoint and we are receiving another pause.
With the fix we don't see the message anymore.